logitech_r400/src/logitech_r400/logitech_r400

```diff
@@ -13,17 +13,13 @@ see README.md
 
 from logitech_r400.msg import LogitechR400
 import rospy
+import rospkg
 from sys import exit as sysexit
 from time import sleep
 import usb.core, usb.util
+import yaml
 
 
-R400_ID_VENDOR = 0x0e00
-R400_ID_PRODUCT = 0x0111
-# we have found 2 different types of values for R400
-R400_ID_VENDOR_2 = 0x046d
-R400_ID_PRODUCT_2 = 0xc538
```

Comment:
Can we add all the known values to the yaml file and then check all of them?
As it is we are forcing the user to "debug" in order to find the IDs that we already know can be associated to the R400